k8s/trillian/README.md

```diff
@@ -0,0 +1,176 @@
+# Overview
```

Comment:
Could you please also add sections about 1) backup & restore, 2) upgrades and 3) uninstallation?

Reply:
Working on adding these sections now.

Reply:
Done.